### DISCUSSION TITLE
Bout variables in ProPASS Wide Format MasterFile

### DISCUSSION THREAD

**Discussion by Suesmith12** [Ergo-Tools/ActiPASS#92] [Help and Q&A (processing, quality checking or statistics)] [answered]
Hi Pasan,

I am using ActiPASS 2024.05. When I generate stats and the ProPASS Wide Format MasterFile is created, I dont see any of the bout variables in this excel file - the variables for time percentiles of continuous segments are there, but I can't see any bout variables for activity/posture/intensity-classes (i.e., those in the attached)
![Screenshot 2024-06-07 122244](https://github.com/Ergo-Tools/ActiPASS/assets/148974338/de3fd90b-fb4f-42cd-9fb3-9ca6038df2bd)? Is there something extra I need to select when generating stats?

I would like to see 1 min and 5 min bouts of INT34 (with ProPASS settings). 

Please let me know if there is something I can do to see this. Thanks, Sue

**Comment by pasan-h** [comment, marked as answer]
Hi Sue,
We decided to disable bout variables in default ProPASS mode because there is a risk of not properly interpreting bout variables (our perception of a bout vs what is provided by ActiPASS). However it is still possible to enable generating those variables. Just open [Advanced settings](https://github.com/Ergo-Tools/ActiPASS/wiki/ActiPASS-FAQ-and-troubleshooting#how-do-i-enable-actipass-advanced-mode-of-processing) and enable bout-variables.
Please be aware of implications we discussed before when you use bout variables😃. 
https://github.com/Ergo-Tools/ActiPASS/discussions/63

In general apart from "SIt+Lie" or "upright" bout variables the rest should be used with a lot of caution.
Best
/Pasan
